PyPi.md

```
@@ -0,0 +1,21 @@
+# Skyline
+
+[![PyPI version](https://badge.fury.io/py/partcad-skyline.svg)](https://pypi.org/project/partcad-skyline/)
+
+![](./docs/assets/images/skyline.png)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)